linetools/isgm/utils.py

```diff
@@ -506,23 +506,25 @@ def synthesize_components(components, zcomp=None, vbuff=0*u.km/u.s):
     # Checks
     assert chk_components(components, chk_A_none=True, chk_match=True)
 
-    # Init final component
-    synth_comp = AbsComponent.from_component(components[0], Ntup=(components[0].flag_N, components[0].logN, components[0].sig_logN))
 
     # Meld column densities
+    obj = dict(flag_N=components[0].flag_N, logN=components[0].logN, sig_logN=components[0].sig_logN)
     for comp in components[1:]:
         if comp.flag_N != 0:
-            synth_comp.flag_N, synth_comp.logN, synth_comp.sig_logN = ltaa.sum_logN(synth_comp, comp)
+            obj['flag_N'], obj['logN'], obj['sig_logN'] = ltaa.sum_logN(obj, comp)
 
-    # Meld z, vlim
     # zcomp
     if zcomp is None:
         zcomp = np.mean([comp.zcomp for comp in components])
-    synth_comp.zcomp = zcomp
+
     # Set vlim by min/max  [Using non-relativistic + buffer]
     vmin = u.Quantity([(comp.zcomp-zcomp)/(1+zcomp)*const.c.to('km/s')+comp.vlim[0] for comp in components])
     vmax = u.Quantity([(comp.zcomp-zcomp)/(1+zcomp)*const.c.to('km/s')+comp.vlim[1] for comp in components])
-    synth_comp.vlim = u.Quantity([np.min(vmin)-vbuff, np.max(vmax)+vbuff])
+    vlim = u.Quantity([np.min(vmin)-vbuff, np.max(vmax)+vbuff])
```

Comment:
Looks like the default value of vbuff is 0 km/s but the docstring indicates 10 km/s?

Reply:
Must have used to have been 10 km/s.  I guess
I figured that was a problem..

Have edited the doc string accordingly and kept it at 0 km/s.